eslint.config.mjs

```diff
@@ -83,6 +85,11 @@ const config = createConfig([
       'jest/prefer-lowercase-title': 'warn',
       'jest/prefer-strict-equal': 'warn',
     },
+    settings: {
```

Comment:
See https://github.com/eslint-community/eslint-plugin-n/blob/86a524250dcc7c32225f2880ec66767a06c6258d/docs/rules/no-unsupported-features/node-builtins.md#shared-settings for why this works.